### PR TITLE
Rx

### PR DESCRIPTION
**Pull request by ArqTras**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -69,7 +69,7 @@ endif()
 include_directories(${Boost_INCLUDE_DIRS})
 
 # include arqma
-include_directories(${ARQMA_SOURCE_DIR}/build/release)
+include_directories(${ARQMA_SOURCE_DIR}/build)
```
Comment (malbit) [inline CMakeLists.txt]: It has to point to /release folder :)

File: cmake/FindArqma.cmake [hunk 2]
```diff
@@ -71,7 +71,8 @@ message(STATUS ${ARQMA_SOURCE_DIR}/build)
 include_directories(
 		${ARQMA_SOURCE_DIR}/src
 		${ARQMA_SOURCE_DIR}/external
-		${ARQMA_SOURCE_DIR}/build/Linux/release-v0.5/release
+		${ARQMA_SOURCE_DIR}/external/randomarq/src
+		${ARQMA_SOURCE_DIR}/build/
```
Comment (malbit) [inline cmake/FindArqma.cmake]: Same as above

File: src/page.h [hunk 3]
```diff
@@ -27,7 +34,16 @@
...
+extern "C" uint64_t rx_seedheight(const uint64_t height);
 
+extern "C" bool rx_needhash(const uint64_t height, uint64_t *seedheight);
+
+// Forked rx-slow-hash.c directly from Arqma
+extern "C" void rx_seedhash(const uint64_t height, const char *hash, const int miners);
+extern "C" void rx_slow_hash(const void *data, size_t length, char *hash, int miners);
+extern "C" void rx_reorg(const uint64_t split_height);
+
+static __thread randomx_vm *rx_vm = NULL;
```
Comment (malbit) [inline src/page.h]: Should be: extern __thread randomx_vm *rx_vm;

File: CMakeLists.txt [hunk 4]
```diff
@@ -6,7 +6,7 @@ set(PROJECT_NAME
 
 project(${PROJECT_NAME})
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14")
+set(CMAKE_CXX_STANDARD 11)
```
Comment (malbit) [inline CMakeLists.txt]: standard 14 includes standard 11 and have newer options so it is even better to have 14 then

**Comment by ArqTras** [comment]
Should be better now

**Pull request closed without merging**